addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -38,11 +23,10 @@
 
     &-top-right {
```

Comment:
This class seems redundant since it's added to the main container by default.
Could you traceback the history of the class and check if it's really needed?